home/screens/FeedbackFormScreen/index.tsx

```diff
@@ -0,0 +1,122 @@
+import { CheckIcon, spacing } from '@expo/styleguide-native';
+import { APIV2Client } from 'api/APIV2Client';
+import {
+  Text,
+  View,
+  TextInput,
+  Button,
+  Heading,
+  Spacer,
+  useExpoTheme,
+} from 'expo-dev-client-components';
+import * as React from 'react';
+import { useState } from 'react';
+import { ActivityIndicator, ScrollView, StyleSheet } from 'react-native';
+
+export function FeedbackFormScreen() {
+  const theme = useExpoTheme();
+  const [submitting, setSubmitting] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
+  const [error, setError] = useState<string>();
+
+  const [feedback, setFeedback] = useState('');
+  const [email, setEmail] = useState('');
+
+  async function onSubmit() {
+    setError(undefined);
+    setSubmitting(true);
+    try {
+      const api = new APIV2Client();
+      await api.sendOptionallyAuthenticatedApiV2Request('/feedback/expo-go-send', {
```

Comment:
```suggestion
      await api.sendUnauthenticatedApiV2Request('/feedback/expo-go-send', {
```

(the endpoint doesn't require/use auth)

Reply:
Updated